install/0000_90_machine-config-operator_01_prometheus-rules.yaml

```diff
@@ -64,7 +64,7 @@ spec:
           labels:
             severity: warning
           annotations:
-            message: "Drain failed on {{ $labels.node }} , updates may be blocked. For more details:  oc logs -f -n {{ $labels.namespace }} {{ $labels.pod }} -c machine-config-daemon"
+            message: "Drain failed on {{ $labels.node }} , updates may be blocked. For more details check MachineConfigController pod logs: oc logs -f -n {{ $labels.namespace }} machine-config-controller-xxxxx -c machine-config-controller"
```

Comment:
To be more helpful, I think it will make sense to also mention `node name` in error since we are adding node name as  prefix in drain log

Reply:
Sorry, to clarify, do you mean to have the nodename as part of this message? I'm pretty sure `{{ $labels.node }}` is the nodename here.

Was thinking how to make this better. I don't know if in prom I can somehow get the MCC name from the MCD view. I was thinking of removing the specific pod command, but adding in some details on how to find the error message. Would that work?